eiffel-vocabulary/EiffelTestExecutionRecipeCollectionCreatedEvent.md

```diff
@@ -67,6 +67,21 @@ __Type:__ Integer
 __Required:__ Yes  
 __Description:__ The execution priority of the batch, as compared to other batches in the collection. A lower value SHALL be interpreted as a higher priority.
 
+#### data.batches.configuration
```

Comment:
As this is intended to keep the same kind of information as data.batches.recipes.constraints, I'd like to see this parameter called data.batches.constraints instead. If you don't think that is suitable you should instead suggest to change the name of the existing parameter (or rather make the current parameter deprecated and propose the new).

Reply:
Thanks for the comment. Renamed the parameter.